benchexec/tools/igml.py

```diff
@@ -0,0 +1,136 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import re
+import sys
+
+from numpy.matlib import identity
+
+import benchexec.result as result
+import benchexec.tools.template
+import benchexec.tools.template
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """Tool info for IGML
+    The tool will be available soon"""
+
+    REQUIRED_PATHS = ["lib/*.jar", "scripts", "igml.jar"]
+
+    def executable(self, tool_locator):
+        executable = tool_locator.find_executable("igml.sh", subdir="scripts")
+        base_dir = os.path.join(os.path.dirname(executable), os.path.pardir)
+        jar_file = os.path.join(base_dir, "igml.jar")
+        bin_dir = os.path.join(base_dir, "target")
+        src_dir = os.path.join(base_dir, "src")
+        return executable
```

Comment:
```suggestion
        return tool_locator.find_executable("igml.sh", subdir="scripts")
```
These variables are never used.